perception/object_recognition/tracking/multi_object_tracker/src/tracker/model/vehicle_tracker.cpp

```diff
@@ -205,13 +207,14 @@ bool VehicleTracker::measure(
 }
 
 bool VehicleTracker::getEstimatedDynamicObject(
-  const ros::Time & time, autoware_perception_msgs::DynamicObject & object)
+  const rclcpp::Time & time, autoware_perception_msgs::msg::DynamicObject & object)
 {
   object = object_;
-  object.id = unique_id::toMsg(getUUID());
+  // TEMP to replace with the uuid generation
+  // object.id = unique_id::toMsg(getUUID());
```

Comment:
Since we solved the generation of uuid, we can now uncomment this correct?
```suggestion
  object.id = getUUID();
```

(and the same for all other places that use getUUID(). )

Reply:
Ah, good catch 9d3c8a1